src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.cs

```diff
@@ -2566,7 +2566,8 @@ private void BuildExtendedProperties(IIndentedStringBuilder outerwriter, XamlObj
 
 					if (_isDebug && IsFrameworkElement(objectDefinition.Type))
 					{
-						writer.AppendLineInvariant($"global::Uno.UI.FrameworkElementHelper.SetBaseUri({closureName}, \"file:///{_fileDefinition.FilePath.Replace("\\", "/")}\");");
+						var lines = $"LN{objectDefinition.LineNumber}_{objectDefinition.LinePosition}";
+						writer.AppendLineInvariant($"global::Uno.UI.FrameworkElementHelper.SetBaseUri({closureName}, \"file:///{_fileDefinition.FilePath.Replace("\\", "/")}#{lines}\");");
```

Comment:
This will break hot reload: https://github.com/unoplatform/uno/blob/7d75ad18dfe7d834e23cd5fc19b83f3e979e6727/src/Uno.UI.RemoteControl/HotReload/ClientHotReloadProcessor.cs#L120

Reply:
It will work only once ;-)  Good catch.